### ISSUE TITLE
config object upgrade to 0.1 does not set 'enabled_plugins' (Trac #30)

### ISSUE THREAD

**Issue by cfobel** [wheeler-microfluidics/microdrop#34]
Migrated from http://microfluidics.utoronto.ca/ticket/30
```json
{
    "status": "closed", 
    "changetime": "2014-04-17T19:39:01", 
    "description": "", 
    "reporter": "cfobel", 
    "cc": "", 
    "resolution": "fixed", 
    "_ts": "1397763541728826", 
    "component": "microdrop", 
    "summary": "config object upgrade to 0.1 does not set 'enabled_plugins'", 
    "priority": "major", 
    "keywords": "", 
    "version": "0.1", 
    "time": "2012-01-04T23:02:54", 
    "milestone": "Microdrop 1.0", 
    "owner": "cfobel", 
    "type": "defect"
}
```

**Comment by cfobel** [comment]
Trac update at 20120104T23:04:45: 
* christian@fobel.net commented:

> commit bf27c2067eb60c1fd65a14963592df85c3dd8992
> Author: Christian Fobel <christian@fobel.net>
> Date:   Wed Jan 4 17:45:11 2012 -0500
> 
>     Update Windows installer to copy plugins...
>     
>     Update Windows installer to copy plugins to common app data directory.
>     Fixes #33.
>     Modify Config.upgrade() to set initialize 'enabled_plugins' to empty list.
>     Fixes #34.


* christian@fobel.net changed resolution from "" to "fixed"
* christian@fobel.net changed status from "new" to "closed"

**Comment by cfobel** [comment]
Trac update at 20140417T19:39:01: ryan changed component from "core" to "microdrop"
